product_tier_validation/data/product_state_data.xml

```diff
@@ -1,8 +1,30 @@
 <odoo noupdate="1">
+    <record id="product_state_cancel" model="product.state">
```

Comment:
I'm always wondering why adding new states here ?

Seems to be project-specific ...

Reply:
Ok, to fit 'cancel' behaviour from base_tier_validation